.github/workflows/release.yaml

```diff
@@ -1,7 +1,11 @@
 name: Build
 
+# Runs every Tuesday at 9 am UTC
 on:
+  schedule:
+    - cron: '0 9 * * 2'
```

Comment:
Just a quesion: for UTC time, 9 am is 1 am in PST, will it be better if it rus at PST's office hours?

Reply:
So we run the submodule sync job https://github.com/runfinch/finch-core/blob/main/.github/workflows/submodulesync.yaml#L5 a 9am UTC as well on Monday. I am okay changing it to ` '0 17 * * 2'`. We also have `workflow_dispatch` as a trigger so incase the oncall has to run it manually, its possible.